swagger.json

```diff
@@ -5200,50 +5200,27 @@
         },
         "DocumentFields": {
             "type": "array",
+            "example": [
+                {"label": "Photos", "valueType": "ValueType_Photo", "photoValue": []},
+                {"label": "Gallons", "valueType": "ValueType_Number", "numberValue": 12.34 },
+                {"label": "City", "valueType": "ValueType_String", "stringValue": "San Francisco, CA" }
+            ],
             "items": {
                 "$ref": "#/definitions/DocumentField"
             }
         },
-        "Document": {
+        "DocumentBase": {
             "type": "object",
             "required": [
-              "driverId",
-              "driverCreatedAtMs",
-              "documentType",
-              "dispatchJobId",
```

Comment:
Removed documentType and dispatchJobId from required, because they are not actually always returned.